src/Pages/SignUp/SignUp.jsx

```diff
@@ -1,8 +1,79 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import SignUpForm from "./components/SignUpForm";
+import "./SignUp.scss";
 
+const API = "http://10.58.7.159:8000/user/signup";
 class SignUp extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userInfo: [],
+    };
+  }
+
+  onSubmitHandler = (
+    email,
+    password,
+    rePassword,
+    loginPath,
+    anotherPath,
+    name
+  ) => {
+    console.log(email, name, password, rePassword, loginPath, anotherPath);
+    fetch(API, {
+      method: "POST",
+      body: JSON.stringify({
+        email: email,
+        password: password,
+        rePassword: rePassword,
+        loginPath: loginPath,
+        anotherPath: anotherPath,
+        name: name,
+      }),
+    })
+      .then((res) => res.json())
+      .then((res) => console.log(res));
+
+    // const updatedUser = [
+    //   ...this.state.userInfo,
+    //   { email, password, rePassword, loginPath, anotherPath },
+    // ];
+    // this.setState({
+    //   userInfo: updatedUser,
+    // });
+  };
+
+  componentDidMount() {
+    // this.state.userInfo &&
+  }
   render() {
-    return <div></div>;
+    console.log(this.state.userInfo);
+    return (
+      <section className="SignUp">
+        <div className="sign-up-container">
+          <h1> 회원가입 </h1>
+          <SignUpForm
+            onSubmitHandler={(
+              email,
+              password,
+              rePassword,
+              loginPath,
+              anotherPath,
+              name
+            ) => {
+              this.onSubmitHandler(
+                email,
+                password,
+                rePassword,
+                loginPath,
+                anotherPath,
+                name
+              );
+            }}
```

Comment:
event 인자 받는 자리라서 지워주세요!
```suggestion
            onSubmitHandler={() => {
              this.onSubmitHandler(
                email,
                password,
                rePassword,
                loginPath,
                anotherPath,
                name
              );
            }}
```

Reply:
아하 여태까지 습관적으로 저렇게 했었네요.